Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -961,7 +990,28 @@ private void MobileDeviceActionHandler(ActMobileDevice act)
                             act.Error = "Operation not supported for this mobile OS or application type.";
                         }
                         break;
-
+                    case ActMobileDevice.eMobileDeviceAction.GetDeviceBattery:
+                        AddReturnParamFromDict(GetDeviceBatteryInfo(), act);
```

Comment:
i'm missing here the implementation of all the methods like "GetDeviceBatteryInfo()" from some reason not seeing it on PR, you will show me on your VS